examples/custom-generator-codelab/src/generators/json.js

```diff
@@ -24,53 +24,53 @@ jsonGenerator.scrub_ = function(block, code, thisOnly) {
   return code;
 };
 
-jsonGenerator['logic_null'] = function(block) {
-  return ['null', jsonGenerator.PRECEDENCE];
+jsonGenerator.forBlock['logic_null'] = function(block, generator) {
+  return ['null', generator.PRECEDENCE];
```

Comment:
I think we should probably switch this over to use an order enum like the other language generators? It seems like having the generator in this codelab work differently would get confusing if folks are looking at this + the code we have in core to help them build their generator.